Library/Homebrew/system_config.rb

```diff
@@ -4,6 +4,7 @@
 require "software_spec"
 require "rexml/document"
 require "development_tools"
+require "extend/ENV"
```

Comment:
Are you sure that's not already included in e.g. `global`?

Reply:
It is not.